src/processinfo.h

```diff
@@ -106,9 +103,40 @@ class ProcessInfo
 
     const string &hostname() const { return _hostname; }
 
-    const UniquePid &upid() const { return _upid; }
+    const UniquePid &upid() {
+      // Temporary fix until we remove the static members from UniquePid.cpp.
```

Comment:
Minor: `UniquePid` is fine for the clarity but `uniquepid.cpp` is the actual name.